bolt-ons/rgb-led-bolt-on/led-bolt-on.ino

```diff
@@ -0,0 +1,88 @@
+#include <Adafruit_NeoPixel.h>
```

Comment:
In my original demo I found the LiteLED library.  Can we used this one instead of the Adafruit one? 
https://docs.arduino.cc/libraries/liteled/

#include <LiteLED.h>

This was my original Demo:
https://autosportlabs.slack.com/archives/C076B8A3DMH/p1729826991072029

My concern with the Adafruit library is it is called "NeoPixel".  Even though it can work with the WS2812, the name is misleading, so I think the  LiteLED is a better library.   (NeoPixels are different types of RGB LED...)